lazer/contracts/sui/sdk/js/src/client.ts

```diff
@@ -0,0 +1,28 @@
+import { bcs } from "@mysten/sui/bcs";
+import { Transaction } from "@mysten/sui/transactions";
+import { SUI_CLOCK_OBJECT_ID } from "@mysten/sui/utils";
+
+const MAX_ARGUMENT_SIZE = 16 * 1024;
+
+export function addParseAndVerifyLeEcdsaUpdateCall(opts: {
+  tx: Transaction;
+  packageId: string;
+  stateObjectId: string;
+  updateBytes: Buffer;
```

Comment:
Note that `Buffer` is not available in browsers without a polyfill, you might want to consider using `Uint8Array` instead which is a superclass of `Buffer` and would make the code more easily isomorphic, if that's something you care about and you aren't using anything specific to `Buffer` (I don't think you are)